client/src/components/Pages/Structure/Input/TemplatedSequenceElementInput/TemplatedSequenceElementInput.tsx

```diff
@@ -21,7 +21,7 @@ interface TemplatedSequenceElementInputProps
 
```

Comment:
Not sure if we want to include it in this ticket, but it'd be great to fix up the strand radio. Right now it pushes the label over in a weird way. Generally the options are positioned and spaced oddly and it'd be nice to keep them compressed in a consistent manner.

<img width="297" alt="Screen Shot 2022-10-19 at 9 32 21 AM" src="https://user-images.githubusercontent.com/12942397/196705772-845c8a5f-774e-4af5-bb62-d977c5e5058b.png">

Reply:
OOP, I should def fix that. I'll work on that now